js/scripts.js

```diff
@@ -1,5 +1,5 @@
 var span = $("span");
-      span.each(function(index, element) {
+span.each(function(index, element) {
 });
 $("span:odd").css('color','red');
 
```

Comment:
where is other part of excercise? you need to append every span in your code by button